kernel/configs/README.md

```diff
@@ -0,0 +1,28 @@
+## How to use defconfig files
+
+defconfig files must be copied in the kernel source code directory at `arch/x86/configs/`
+and then using `make` command the `.config` file can be generated.
+
+For example
+
+```
+cp x86_kvm_4.14.x_defconfig linux-4.14.22/arch/x86/configs/
+popd linux-4.14.22
+make ARCH=x86 x86_kvm_4.14.x_defconfig
```

Comment:
this is due to the config name, should we move the config name to include the prefix `kata`?
I am not familiar using the ARCH env variable Also are you testing this kernel with ARCH=x86 should not be x86_64?

Reply:
@jcvenegas `ARCH=x86` works for me, but I see in cc packaging we use x86_64, let me change it, thanks